.github/workflows/release-ci.yml

```diff
@@ -30,101 +30,14 @@ on:
-        env:
-          # Inputs
-          DRY_RUN: ${{ github.event.inputs.dryRun == 'true' && 'true' || '' }}
-          FORCE_INTEGRATION_RELEASE: ${{ github.event.inputs.forceIntegrationRelease == 'true' && 'true' || '' }}
-          # Other
-          GITHUB_CONTEXT: ${{ toJson(github) }}
-          # https://docs.npmjs.com/generating-provenance-statements
-          NPM_CONFIG_PROVENANCE: true
-          # Secrets
-          # Note: must use personal access token
-          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
-          REDIS_URL: ${{ secrets.REDIS_URL }}
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-          SLACK_RELEASE_FEED_WEBHOOK: ${{ secrets.SLACK_RELEASE_FEED_WEBHOOK }}
-
-      - name: Print output
-        env:
-          THE_OUTPUT: '${{ toJson(steps.publish.outputs) }}'
-        run: |
-          echo "$THE_OUTPUT"
-
-      - name: Create a tag on prisma repository
-        id: tag-prisma
-        if: ${{ steps.publish.outputs.prismaVersion != '' && !(github.event_name == 'workflow_dispatch' && github.event.inputs.dryRun == 'true') }}
-        uses: ./.github/actions/create-git-tag
-        with:
-          token: ${{ secrets.BOT_TOKEN }}
-          tag-name: ${{ steps.publish.outputs.prismaVersion }}
-          commit-sha: ${{ github.sha }}
-
-      - name: Create a tag on prisma-engines repository
-        uses: ./.github/actions/create-git-tag
-        id: tag-prisma-engines
-        if: ${{ steps.publish.outputs.prismaVersion != '' && !(github.event_name == 'workflow_dispatch' && github.event.inputs.dryRun == 'true') }}
-        with:
-          token: ${{ secrets.BOT_TOKEN }}
-          repository: prisma/prisma-engines
-          tag-name: ${{ steps.publish.outputs.prismaVersion }}
-          commit-sha: ${{ steps.publish.outputs.enginesCommitHash }}
-
-  # We also have `sendSlackMessage()` in publish.ts
-  # It uses the #feed-prisma-releases channel and adds more information
-  success:
-    needs:
-      - release
-    if: ${{ success() }}
-    name: Communicate success
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set current job url in SLACK_FOOTER env var
-        run: echo "SLACK_FOOTER=<$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID|Click here to go to the job logs>" >> "$GITHUB_ENV"
-
-      - name: Slack Notification on Success
-        uses: rtCamp/action-slack-notify@v2.3.2
-        env:
-          SLACK_TITLE: 'prisma/prisma Release ${{ needs.release.outputs.prismaVersion }} succeeded :white_check_mark:'
-          SLACK_COLOR: '#55ff55'
-          SLACK_WEBHOOK: ${{ secrets.SLACK_RELEASE_FEED_WEBHOOK }}
-          SLACK_CHANNEL: feed-prisma-publish
+    uses: ./.github/workflows/release.yml
+    with:
+      timeout-minutes: 75
+      dry-run: ${{ github.event.inputs.dryRun == 'true' }}
+      force-integration-release: ${{ github.event.inputs.forceIntegrationRelease == 'true' }}
+      skip-tagging-on-dry-run: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
I'm actually not sure it is still necessary, I didn't touch it when adding `prisma/prisma` tags (`prisma-engines` tagging step already had these conditions as they were) but I feel like it's unnecessary (especially now it's obvious because your refactoring made the logic easier to understand). We should not be creating git tags if we're not publishing the packages to npm, regardless of the event which triggered this. Besides, a dry run implies `workflow_dispatch` anyway because we don't do dry runs automatically.